scripts/redirects.js

```diff
@@ -0,0 +1,83 @@
+function globToRegex(glob) {
+  return new RegExp(`^${glob.replace(/\*/g, '(.*)').replace(/\?/g, '(.)').replace(/\//g, '\\/')}$`);
```

Comment:
This would only ever be an issue if the authors add backslashes in the smart-redirects (specifically the `from` column in the excel) which would not be the case. everything is forward slash in the web thankfully.